src/Monolog/Handler/ChromePHPHandler.php

```diff
@@ -163,15 +162,28 @@ protected function send(): void
                 'extra' => [],
             ];
             self::$json['rows'][count(self::$json['rows']) - 1] = $this->getFormatter()->format($record);
-            $json = Utils::jsonEncode(self::$json, null, true);
-            $data = base64_encode(utf8_encode($json));
+            $data = $this->encodeArray();
         }
 
         if (trim($data) !== '') {
             $this->sendHeader(static::HEADER_NAME, $data);
         }
     }
 
+    /**
+     * Encode and format json
+     */
+    private function encodeArray(): string
+    {
+        $json = Utils::jsonEncode(self::$json, Utils::DEFAULT_JSON_FLAGS & ~JSON_UNESCAPED_UNICODE, true);
+
+        return base64_encode(
+            function_exists('mb_convert_encoding')
+            ? mb_convert_encoding($json, 'UTF-8', mb_list_encodings())
+            : utf8_encode($json)
+        );
```

Comment:
I think we can actually skip the encoding here, as json_encode will produce utf-8 or fail since json must be utf-8. And we also have JSON_INVALID_UTF8_SUBSTITUTE in the Utils::DEFAULT_JSON_FLAGS now so it is really useless I think.

Anyway I'll fix the patch.